packages/create-webpack-app/templates/init/react/index.html.tpl

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Webpack + React</title>
+        <title>React Webpack Project</title>
```

Comment:
small w in webpack

Reply:
I will make it consistent across the whole package, in next commit